AsyncDisplayKit/ASImageNode.mm

```diff
@@ -110,17 +109,11 @@ - (instancetype)initWithViewBlock:(ASDisplayNodeViewBlock)viewBlock didLoadBlock
   return nil;
 }
 
-- (void)dealloc
```

Comment:
@johnepinterest: `-dealloc` was added back in before it was merged. If that's the only difference then we are good.